recipes/span-lite/all/conanfile.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
+from conans import ConanFile, tools
+
+
+class SpanLite(ConanFile):
+    name = "span-lite"
+    
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/martinmoene/span-lite"
+    description = "span lite - A C++20-like span for C++98, C++11 and later in a single-file header-only library"
+    author = "Martin Moene"
```

Comment:
Author attributes indicate the author of the recipe. As Conan Center Index is a collaborative project, recipes will be maintained and improved by the community, so we feel that the authorship will be tracked in the github history of the repo
```suggestion
```